packages/icons/src/types.ts

```diff
@@ -60,6 +60,8 @@ const icons = [
 	'change-history',
 	'chart-area',
 	'chart-bar',
+	'chart-cube-root',
+	'chart-crossing-lines',
```

Comment:
nit: but maybe call this `chart-lines-crossing` to align with our other `chart-line-*` icons

Reply:
that makes a lot more sense, thank you!